src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,267 @@
+package duke;
+
+import duke.exception.InvalidCommandException;
+import duke.exception.InvalidDeadlineException;
+import duke.exception.InvalidDescriptionException;
+import duke.exception.InvalidEventException;
+import duke.exception.InvalidNumberException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.task.ToDo;
+
+/**
+ * This class deals with making sense of the user command.
+ */
+public class Parser {
+    private Ui ui;
+
+    public Parser() {
+        ui = new Ui();
+    }
+
+    /**
+     * Processes the user commands and decides the correct output.
+     *
+     * @param command The user command.
+     * @param taskList The user's list of tasks.
+     * @param storage Storage of the user's task list.
+     * @return String output depending on the input command.
+     */
+    public String processCommand(String command, TaskList taskList, Storage storage) throws Exception {
+        storage.createFile();
+
+        if (command.equals("start")) {
+            return ui.startMessage();
+        }
+
+        if (command.equals("list")) {
```

Comment:
Perhaps you could use a `switch` statement here instead of multiple `if-else-` clauses?